boa/src/builtins/value/mod.rs

```diff
@@ -767,7 +767,7 @@ pub(crate) fn log_string_from(x: &ValueData, print_internals: bool) -> String {
 }
 
 /// A helper function for specifically printing object values
-fn display_obj(v: &ValueData, print_internals: bool) -> String {
+pub fn display_obj(v: &ValueData, print_internals: bool) -> String {
```

Comment:
I would restrict this to the crate:
```suggestion
pub(crate) fn display_obj(v: &ValueData, print_internals: bool) -> String {
```